app/sidekiq/decision_review/delete_saved_claim_records_job.rb

```diff
@@ -12,6 +12,8 @@ class DeleteSavedClaimRecordsJob
     STATSD_KEY_PREFIX = 'worker.decision_review.delete_saved_claim_records'
 
     def perform
+      Rails.logger.warn("#{self.class.name} job is deprecated and will be replaced by DR engine job")
```

Comment:
Could we use `ActiveSupport::Deprecation.new#warn` here and in the other jobs?

e.g.

```  ActiveSupport::Deprecation.new.warn('... job is deprecated...')
```

Reply:
PR updated to use `ActiveSupport::Deprecation.new#warn`